src/framework/theme/components/icon/icon-pack.ts

```diff
@@ -0,0 +1,23 @@
+import { NbIcon } from './icon';
+
+export interface NbIcons {
+  [key: string]: NbIcon | string;
+}
+
+export enum NbIconPackType  {
+  SVG = 'svg',
+  FONT = 'font',
+}
+
+export interface NbIconPackParams {
+  packClass?: string,
```

Comment:
`iconElementClass`? Not sure about this
Also it's unclear, should I prefix packClass when iconPrefix specified.